src/qt/transactiontablemodel.cpp

```diff
@@ -385,28 +365,33 @@ QString TransactionTableModel::formatTxType(const TransactionRecord *wtx) const
     case TransactionRecord::SendToSelf:
         return tr("Payment to yourself");
     case TransactionRecord::Generated:
-	    if (wtx->RemoteFlag==1 && false)
-		{
-
-			double reward = CoinToDouble(wtx->credit + wtx->debit);
-			double max = GetMaximumBoincSubsidy(GetAdjustedTime());
-			if (reward==max)
-			{
-				return tr("Mined - DPOR");
-			}
-			else
-			{
-				return tr("Minted - (Local) DPOR");
-			}
-		}
-		else if (((IsPoR(CoinToDouble(wtx->credit + wtx->debit)))))
-		{
-				return tr("Mined - PoR");
-		}
-		else
-		{
-				return tr("Mined - Interest");
-		}
+        if (wtx->RemoteFlag==1 && false)
```

Comment:
Looks like we can nuke this section since it's not executed.

Reply:
10-4